source/installation/config.html.md

```diff
@@ -43,18 +44,41 @@ MAIL_USER=example
 MAIL_PASSWORD=123456
 MAIL_HELO_NAME=example.org
 MAIL_ADDRESS=from@example.org
+
+# Custom icons
+#PLUME_LOGO=icons/custom/plume.png
+#PLUME_LOGO_FAVICON=icons/custom/plume32.png
+#PLUME_LOGO_48=icons/custom/plume48.png
+#PLUME_LOGO_72=icons/custom/plume72.png
+#PLUME_LOGO_96=icons/custom/plume96.png
+#PLUME_LOGO_144=icons/custom/plume144.png
+#PLUME_LOGO_160=icons/custom/plume160.png
+#PLUME_LOGO_192=icons/custom/plume192.png
+#PLUME_LOGO_256=icons/custom/plume256.png
+#PLUME_LOGO_512=icons/custom/plume512.png
```

Comment:
I'm not sure we want to add that, this was meant to be a "minimal" config example IMO. There is a link for all the options just under, for people who want deep customization.

Reply:
Fair enough. I agree, but wanted to make sure.